src/core/pageModules/CardsModule/CardsModule.tsx

```diff
@@ -32,18 +32,26 @@ export function CardsModule({ items }: CardsModuleProps) {
         items.length === 1 && styles.singleGridWrapper,
       )}
     >
-      {items?.map((card) => (
-        <SimpleCard
-          title={card.title}
-          description={card.description}
-          linkTarget={card.link.target}
-          linkTitle={card.link.title}
-          linkUrl={card.link.url}
-          backgroundColor={card.backgroundColor}
-          direction={items.length === 1 ? 'horisontal' : 'vertical'}
-          icon={card.icon}
-        />
-      ))}
+      {items?.map((card, index) => {
+        // The card module does not contain any proeprty that could be used as an unique id, so one needs to be created
+        const uniqueKey = `card-${index}-${card.title?.substring(
+          0,
+          4,
+        )}-${card.description?.substring(0, 4)}`;
```

Comment:
How about making this more robust by using a hash of the tuple of the input fields that should together suffice to make the key unique—hashing only to make the result smaller, the uniqueness property should come from the tuple of the selected input fields (e.g. in [sports](https://github.com/City-of-Helsinki/events-helsinki-monorepo/blob/sports-helsinki-v1.12.0/apps/sports-helsinki/src/domain/venue/venueHighlights/VenueHighlights.tsx#L18-L23))?

For example:
```typescript
import { createHash } from 'crypto';
...
const uniqueKey = createHash('sha256').update(`${card.title}: ${card.description}`).toString();
```

Reply:
yea, good thing. I actually thought about it, but I also thought that the crypto-module would be an external lib. But it seems to be available.